src/sizes/retained_size_and_held_objects.h

```diff
@@ -15,9 +15,11 @@ class RetainedSizeAndHeldObjectsAction : public MemoryAgentTimedAction<jobjectAr
 
     jvmtiError estimateObjectSize(jobject &object, jlong &retainedSize, std::vector<jobject> &heldObjects);
 
-    jvmtiError retagStartObject(jobject object);
-
     jobjectArray createResultObject(jlong retainedSize, jlong shallowSize, const std::vector<jobject> &heldObjects);
+
+    jvmtiError traverseHeapForTheFirstTime(jobject &object);
```

Comment:
Aren't these methods implementation details and shouldn't be a part of public API?

Reply:
These methods are private